### PR TITLE
Issue 103 navbar

### PR DESCRIPTION
**Pull request by Heena31**
## Describe your changes

- Navbar Updated
- Background Color is updated

## Screenshots - If Any (Optional)

##Before

image

##After

image

## Issue ticket number and link - If Any

- 103 

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Starred the repository.
- [x] Added My Name to the Contributor's List.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.

## Additional Information (Optional)
Any additional information that you want to give us

### CODE REVIEW

**Comment by Heena31** [comment]
@anmode Kindly check this is done :)

File: components/common/Header/config.js [hunk 1]
```diff
@@ -40,54 +40,25 @@ export const icons = [
...
-    backgroundColor: '#c099ff',
-    backgroundImage: `radial-gradient(at 2% 10%, hsla(285,85%,79%,1) 0px, transparent 50%),
-  radial-gradient(at 83% 15%, hsla(332,95%,65%,1) 0px, transparent 50%),
-  radial-gradient(at 63% 5%, hsla(239,80%,68%,1) 0px, transparent 50%),
-  radial-gradient(at 0% 61%, hsla(328,80%,77%,1) 0px, transparent 50%),
-  radial-gradient(at 39% 31%, hsla(356,80%,70%,1) 0px, transparent 50%),
-  radial-gradient(at 92% 33%, hsla(344,84%,70%,1) 0px, transparent 50%),
-  radial-gradient(at 9% 14%, hsla(197,61%,68%,1) 0px, transparent 50%);`
+  backgroundColor: 'rgb(120,2,6)',
+  backgroundImage: `linear-gradient(99deg, rgba(120,2,6,1) 21%, rgba(64,5,33,1) 49%, rgba(21,27,69,1) 89%);`
```
Comment (anmode) [inline components/common/Header/config.js:44]: why this change??

File: components/common/Header/config.js [hunk 2]
```diff
@@ -40,54 +40,25 @@ export const icons = [
...
-    backgroundImage: `radial-gradient(at 27% 21%, hsla(310,75%,73%,1) 0px, transparent 50%),
-    radial-gradient(at 65% 82%, hsla(266,61%,71%,1) 0px, transparent 50%),
-    radial-gradient(at 21% 49%, hsla(343,84%,72%,1) 0px, transparent 50%),
-    radial-gradient(at 84% 92%, hsla(359,78%,73%,1) 0px, transparent 50%),
-    radial-gradient(at 39% 54%, hsla(90,90%,69%,1) 0px, transparent 50%),
-    radial-gradient(at 38% 13%, hsla(97,93%,70%,1) 0px, transparent 50%),
-    radial-gradient(at 93% 52%, hsla(81,85%,78%,1) 0px, transparent 50%);`
+    backgroundColor: 'rgb(196,88,153);',
+    backgroundImage: `linear-gradient(195deg, rgba(196,88,153,1) 19%, rgba(83,14,115,1) 56%, rgba(0,0,0,1) 98%, rgba(173,83,137,1) 100%);`
+
```
Comment (anmode) [inline components/common/Header/config.js:49]: same here

File: package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.21",
     "mongoose": "^6.2.7",
-    "next": "^12.3.3",
+    "next": "^12.3.4",
```
Comment (anmode) [inline package.json:21]: please revert this! We do not want to update the package. as of now

File: package.json [hunk 4]
```diff
@@ -35,6 +35,7 @@
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-next": "12.0.7",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^2.26.0"
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-prettier": "^4.2.1"
```
Comment (anmode) [inline package.json:39]: revert this too!

**Review by anmode — CHANGES_REQUESTED** [review]
Nice left some comments...Thanks

File: components/common/Header/config.js (hunk 1 above)
Comment (Heena31) [reply components/common/Header/config.js:44]: Before background color was not matching with the navbar so i added bright colors…

File: package.json (hunk 3 above)
Comment (Heena31) [reply package.json:21]: Sure !! @anmode

**Comment by Heena31** [comment]
@anmode Can you check if it is working fine

**Review by anmode — APPROVED** [review]
Looks cool! one LGTM from any mentor. I'll merge then

**Comment by anmode** [comment]
@Heena31 Could you please attach pic for mobile view too? as we want to see is it mobile optimized or not.

**Comment by Heena31** [comment]
image


@anmode Check this

File: components/common/Header/config.js (hunk 1 above)
Comment (anmode) [reply components/common/Header/config.js:44]: @Heena31... please could you try to reuse our color scheme..I think it is good to have the previous one. 
adding is ok...but for removing some parts there should be some correct logic.I do not found it!

I'll check on my local so I can provide some detailed comments..till the time you can try to reuse the previous scheme

File: styles/Navbar.module.css [hunk 5]
```diff
@@ -212,13 +218,14 @@ navigation menu opens in mobile phones */
...
+  
 
   .nav_active.active {
     color: white;
     text-shadow: none !important;
   }
 
   .navList_active {
-    background-color: rgb(62, 62, 252);
+    background-color: rgb(153, 120, 223);
```
Comment (anmode) [inline styles/Navbar.module.css:229]: This is ok! you added bright colors

**Review by anmode — CHANGES_REQUESTED** [review]
Need a review

File: styles/Navbar.module.css (hunk 5 above)
Comment (Heena31) [reply styles/Navbar.module.css:229]: @anmode definitely i will check

Comment (anmode) [reply styles/Navbar.module.css:229]: that's cool! waiting for your next commit

**Comment by AmanJain18** [comment]
I will upadate here again, as some conflict in these files compared to cureent new one... So currently hold the PR as of Now...

**Comment by anmode** [comment]
@Heena31 ? are you working

**Comment by Heena31** [comment]
> @Heena31 ? are you working

Yay, but I think this program has been done by someone else !! @anmode
